burnman/data/input_seismic/prem.txt

```diff
@@ -0,0 +1,95 @@
+#Depth(m)	Radius(m)	Pressure(Pa)	Gravity(m/s^2)	Density(g/m^3)	Vp(m/s)	Vs(m/s)	Qk	Qmu
```

Comment:
I don't like giving discontinuities a finite width - PREM is no longer PREM in this case. Would it be better to have a line in the scripts that imposes a small dz (dr, dp) when z[i] = z[i+1]? Something like this, perhaps:
# Create interpolatable depths list
for i, depth in enumerate(depths):
    if depth <= depths[i-1]:
        depth += 1e-12